package.json

```diff
@@ -16,6 +17,7 @@
     "react-dom": "^18"
   },
   "devDependencies": {
+    "@antfu/ni": "^0.21.12",
```

Comment:
Are we OK with this dep? I just want to be super super cautious with adding anythign